src/status_im2/contexts/chat/events.cljs

```diff
@@ -294,20 +297,70 @@
 
 (rf/defn mute-chat-toggled-successfully
   {:events [:chat/mute-successfully]}
-  [{:keys [db]} chat-id muted-till]
+  [{:keys [db]} chat-id muted-till mute-type muted? chat-type]
   (log/debug "muted chat successfully" chat-id " for" muted-till)
-  {:db (assoc-in db [:chats chat-id :muted-till] muted-till)})
+  (let [time-string (fn [duration-kw unmute-time]
+                      (str (i18n/label duration-kw)
```

Comment:
Ideally we would create a new translation to avoid concatenating strings like this.